pkgs/tools/security/trousers/allow-non-tss-config-file-owner.patch

```diff
@@ -7,10 +7,10 @@ diff -ur trousers-0.3.11.2.orig/src/tcsd/tcsd_conf.c trousers-0.3.11.2/src/tcsd/
  
 +#ifndef ALLOW_NON_TSS_CONFIG_FILE
  	/* make sure user/group TSS owns the conf file */
- 	if (pw->pw_uid != stat_buf.st_uid || grp->gr_gid != stat_buf.st_gid) {
+ 	if (stat_buf.st_uid != 0 || grp->gr_gid != stat_buf.st_gid) {
  		LogError("TCSD config file (%s) must be user/group %s/%s", tcsd_config_file,
 @@ -775,6 +776,7 @@
- 		LogError("TCSD config file (%s) must be mode 0600", tcsd_config_file);
+ 		LogError("TCSD config file (%s) must be mode 0640", tcsd_config_file);
```

Comment:
I feel like this should be upstreamed, there's nothing nixos specific

Reply:
Agreed.
It looks like it's this way since the introduction of the package in nixpkgs (#2129). Maybe @alexanderkjeldaas knows why the patch has not been upstreamed?

This probably should be not a blocker to pull in the fixes for the security issues though.

Reply:
if this is exposed in a module, we can align the user so that it's the same when being used.